src/modules/CaaSMapper.ts

```diff
@@ -127,6 +144,7 @@ export class CaaSMapper {
         `Item with identifier '${identifier}' was tried to register from remoteProject '${remoteProjectId}' but no remote key was found in the config.`
```

Comment:
we could normalize identifiers here to ensure they are always of the form {id}.{locale}
Either that, or only use {id} for reference management. Using them both only adds confusion & complexity